File: codefac-web/src/main/webapp/resources/js/ejemplo.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

function play() {
    var audio = document.getElementsByTagName("audio")[0];
    //audio.load()
    audio.play();
    var display = document.getElementsByTagName("input")[0];
    display.value = audio.src;
    alert(audio.src);

}
function pause() {
    var audio = document.getElementsByTagName("audio")[0];
    audio.pause();
}

function loadSrc(path)
{
    var audio = document.getElementsByTagName("audio")[0];
    audio.src = path;
    //audio.src=;
}

function imprimir()
{

    var dato = document.getElementsByTagName("formPrueba:dato");
    alert(dato.value);
}

function getVars() {
    // ...
    //alert('aqui');
    
    var x = 12;
    var y = 24;
    
        
    document.getElementById("formId:x").value = x;
    document.getElementById("formId:y").value = y;
    
    alert(x);
}

File: codefac-web/src/main/java/ec/com/codesoft/web/operador/Ejemplo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package ec.com.codesoft.web.operador;

import java.io.Serializable;
import javax.annotation.PostConstruct;
import javax.faces.bean.ManagedBean;
import javax.faces.bean.ViewScoped;
import org.primefaces.context.RequestContext;

/**
 *
 * @author Suco
 */
@ManagedBean
@ViewScoped
public class Ejemplo implements Serializable{
    
    private String focus;
    
    @PostConstruct
    private void init()
    {
        focus="formulario:campo1";
        RequestContext.getCurrentInstance().execute("PF('"+focus+"').show()");
    }
    
    public void imprimir(){
        System.out.println("JJJJJJJJJ");
    }
    
    public void cambiarfocus(String id)
    {
        System.out.println("cambiando focus");
        focus=id;
        RequestContext.getCurrentInstance().execute("focusField('123');");
       
    }

    public String getFocus() {
        return focus;
    }

    public void setFocus(String focus) {
        this.focus = focus;
    }
    
    
    
}
